bot/exts/info/information.py

```diff
@@ -42,15 +43,29 @@ def get_channel_type_counts(guild: Guild) -> DefaultDict[str, int]:
         return channel_counter
 
     @staticmethod
-    def get_member_counts(guild: Guild) -> Dict[str, int]:
+    def join_role_stats(role_ids: list[int], guild: Guild, name: Optional[str] = None) -> dict[str, int]:
+        """Return a dictionary with the number of `members` of each role given, and the `name` for this joined group."""
+        members = 0
+        for role_id in role_ids:
+            if (role := guild.get_role(role_id)) is not None:
+                members += len(role.members)
+            else:
+                raise NonExistentRoleError(role_id)
+        return {name or role.name.title(): members}
+
+    @staticmethod
+    def get_member_counts(guild: Guild) -> dict[str, int]:
         """Return the total number of members for certain roles in `guild`."""
-        roles = (
-            guild.get_role(role_id) for role_id in (
-                constants.Roles.helpers, constants.Roles.mod_team, constants.Roles.admins,
-                constants.Roles.owners, constants.Roles.contributors,
-            )
+        role_ids = [constants.Roles.helpers, constants.Roles.mod_team, constants.Roles.admins,
+                    constants.Roles.owners, constants.Roles.contributors]
+
+        role_stats = {}
+        for role_id in role_ids:
+            role_stats.update(Information.join_role_stats([role_id], guild))
+        role_stats.update(
+            Information.join_role_stats([constants.Roles.project_leads, constants.Roles.domain_leads], guild, "Leads")
```

Comment:
Elsewhere we use `self` from what I found (instead of `Information`).

Reply:
But it's a static method, and the reason it is a static method is because it's a helper function, but it is sort of unique to this cog, so I put it directly into this cog.

Reply:
It's still a method, see the following examples
https://github.com/python-discord/bot/blob/4aaf2bbe3f0a866451cf9fc91e771dfd7c54d029/bot/converters.py#L91-L92
https://github.com/python-discord/bot/blob/4aaf2bbe3f0a866451cf9fc91e771dfd7c54d029/bot/converters.py#L107
...
https://github.com/python-discord/bot/blob/4aaf2bbe3f0a866451cf9fc91e771dfd7c54d029/bot/exts/moderation/silence.py#L367-L368
https://github.com/python-discord/bot/blob/4aaf2bbe3f0a866451cf9fc91e771dfd7c54d029/bot/exts/moderation/silence.py#L182

Reply:
In the examples they might have been called with `self` but the context was different. In case of `silence` for example it was called directly there (`async def silence(...)`) which in fact is not a staticmethod but a normal method. But in my case, I'm calling a staticmethod inside a staticmethod.(Which in fact is strange, but I could not put it into the helpers unfortunately, so I put it directly into the cog as a staticmethod.).
But if you prefer those staticmethod decorators to be removed I'll do so. (Although originally the `get_member_counts` was a staticmethod as well)

Reply:
Oh sorry, I see now. You do not have `self` as an argument